src/seedu/addressbook/commands/FindByTagCommand.java

```diff
@@ -0,0 +1,68 @@
+package seedu.addressbook.commands;
+
+import java.util.*;
```

Comment:
remember to disable intellij auto import wildcard
[Why is wild card import bad?](https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad)